frontend/src/Editor/Comments.jsx

```diff
@@ -1,16 +1,16 @@
 import '@/_styles/editor/comments.scss';
-
+import { shallow } from 'zustand/shallow';
 import React from 'react';
 import { isEmpty } from 'lodash';
-
 import Comment from './Comment';
 import { commentsService } from '@/_services';
-
+import { useAppVersionStore } from '@/_stores/appVersionStore';
 import useRouter from '@/_hooks/use-router';
 
-const Comments = ({ newThread = {}, appVersionsId, socket, canvasWidth, currentPageId }) => {
+const Comments = ({ newThread = {}, socket, canvasWidth, currentPageId }) => {
   const [threads, setThreads] = React.useState([]);
   const router = useRouter();
+  const { appVersionsId } = useAppVersionStore((state) => ({ appVersionsId: state?.editingVersion?.id }), shallow);
```

Comment:
do we need shallow here.? 
const appVersionsId = useAppVersionStore((state) => state?.editingVersion?.id);

We need to return only the version id string here right.?

Reply:
yes, We won't need a shallow here since this is only a string. Kept it in case if we"ll derive more states objects from useAppVersionStore, But yeah we can add it when needed. Removing it.